src/test/java/com/yegor256/JhomeTest.java

```diff
@@ -101,6 +103,23 @@ void findsJavacOnAnyOs() throws IOException {
         );
     }
 
+    @Test
+    void throwsIfJavacNotFound(final @TempDir Path temp) {
+        final Jhome jhome = new Jhome(temp);
+        final IllegalStateException exception = Assertions.assertThrows(
```

Comment:
@volodya-lombrozo see this: https://www.yegor256.com/2015/09/01/redundant-variables-are-evil.html